corehq/apps/data_analytics/models.py

```diff
@@ -15,7 +15,8 @@ class MALTRow(models.Model):
     user_id = models.TextField()
     username = models.TextField()
     email = models.EmailField()
-    is_web_user = models.BooleanField(default=False)
+    user_type = models.TextField()
```

Comment:
My idea was to make this a SmallIntField with mapping like I did for wam/pam. But I kept it like this for now - so as to observe any unknown scenarios of usertype from the datasource, After few runs and observing data, I am planning to change it to a predefined maping